client/signup/steps/woocommerce-install/complete.tsx

```diff
@@ -0,0 +1,34 @@
+import { BackButton, NextButton, SubTitle, Title } from '@automattic/onboarding';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement } from 'react';
+import { GoToStep } from '../../types';
+
+interface Props {
+	goToStep: GoToStep;
+}
+
+export default function Complete( { goToStep }: Props ): ReactElement | null {
+	const { __ } = useI18n();
```

Comment:
I'm not sure if `@wordpress/react-i18n` is set up in the Calypso environment. It'd be cool if it is. We've been using the classic `translate()` function from `i18n-calypso`.

Reply:
Just tested it out. `@wordpress/react-i18n` totally works. That's good to know!